src/client/src/pages/detail/attachments/fileDropzone.jsx

```diff
@@ -13,9 +13,10 @@ import { theme } from "../../../AppTheme.ts";
  * @param {string} props.defaultText - The default text to display in the dropzone.
  * @param {number} props.maxFilesToSelectAtOnce - The maximum number of files that can be selected at once.
  * @param {number} props.maxFilesToUpload - The maximum number of files that can be uploaded.
- * @param {boolean} props.restrictAcceptedFileTypeToCsv - Whether to restrict accepted file types to CSV.
+ * @param {Array<string>} props.acceptedFileTypes - The list of accepted file types.
  * @param {boolean} props.isDisabled - Whether the dropzone is disabled.
  * @param {string} props.dataCy - The data-cy attribute for testing.
+ * @param {Function} props.setFileType - A callback function to set the file type.
```

Comment:
PP: Entspricht so schon dem typescript typ

```suggestion
 * @param {Function} props.setFileType - A react SetStateAction to set the file type.
```